code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -991,6 +991,16 @@
 		/mob/living/carbon/human/proc/change_monitor
 		)
 
+/datum/species/machine/handle_life(mob/living/carbon/human/H)
```

Comment:
I don't like how this is handled - it might be better to add an `on_life` for MMI holders that checks for the presence of a cell mount and its charge, and then call `death()` if there's no power available.
Then, you can add a proc, `is_autorevivable`, to brain organs - and override it on the MMI to carry over the code found [here](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/carbon/human/human_damage.dm#L20) - as well as the check for a cell mount and power being available. That way, we can axe the `can_revive_by_healing` species flag entirely, and have it implemented entirely on organs.